<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpServer</class>
 <widget class="QDialog" name="TcpServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>端口号:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QListWidget" name="ContentListWidget">
      <property name="maximumSize">
       <size>
        <width>1770</width>
        <height>450</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QPushButton" name="createPushButton">
      <property name="text">
       <string>创建聊天室</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>                                  TCP Server</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="portLineEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
